README.md

```diff
@@ -4,135 +4,68 @@
-up a proper build enviornment for CUDA support.
-
-Open-CE expects the `CUDA_HOME` environment variable to be set to the location of the CUDA installation. Note that not all recipes work when `CUDA_HOME` references a non-standard CUDA installation location. Reference the [cuda README](doc/README.cuda_support.md) for more information.
+This is the Open-CE project for feedstock collection, environment data, and build scripts
 
-When building packages that use CUDA, a tar package of TensorRT for the intended CUDA version will need to be [downloaded](https://developer.nvidia.com/nvidia-tensorrt-7x-download) ahead of time. The downloaded file should be placed in a new local directory called `local_files`. The [cuda README](doc/README.cuda_support.md) has more information.
+Welcome to the **open-ce** project. The project contains everything that is needed to build conda packages for
+a collection of machine learning and deep learning frameworks. All packages created for a specific version of
+Open-CE have been designed to be installed within a single conda environment. For more information on `conda`,
+please look at conda's official [documentation](https://docs.conda.io/).
 
-### Building a Collection of Packages
+This repository contains a collection of Open-CE files that can be used to create a conda channel. The conda channel
+will contain packages for every feedstock listed within the Open-CE files. Different variants of Python and CUDA can
+be specified at build time. Open-CE currently supports the following:
 
-To build an entire integrated and functional conda channel using Open-CE, start by installing the needed tools in the [Requirements](#requirements) section above.
-The `open-ce build env` command can then be used to build a collection of Open-CE packages. An Open-CE environment file needs to be passed in as input. A selection of environment files are provided within the `envs` directory for different frameworks such as TensorFlow and PyTorch. The output from running `open-ce build env` will be a local conda channel (by default called `condabuild`) and one or more conda environment file(s) in the output folder depending on the selected build configuration. For more details on `open-ce build env`, please see [`doc/README.open_ce_build.md`](doc/README.open_ce_build.md#open-ce-build-env-sub-command).
-
-The following commands will use the `opence-env.yaml` Open-CE environment file to build all of the Open-CE packages for Python 3.6 (the default), including CUDA builds and cpu-only builds (also the default). The commands should be run from within the same directory that contains `local_files`.
-
-```bash
-# Clone Open-CE from GitHub
-git clone https://github.com/open-ce/open-ce.git
-# Build packages
-./open-ce/open-ce/open-ce build env open-ce/envs/opence-env.yaml
-```
+| | Supported Versions |
+| --- | --- |
+| Architecture | Power, x86 |
```

Comment:
I kind of always wanted to change this to:
```
linux-64, linux-ppc64le
```
or
```
x86-64, ppc64le
```

But this matches whats on the 1.2 branch for now, so maybe we just fix that in a separate update.

Reply:
Yeah, we should probably only make readme and doc change sin the main branch.